hack/tools/update_providers/README.md

```diff
@@ -0,0 +1,53 @@
+# Update CAPI Providers with CAPI beta releases
+
+`provider_issues` is a go utility intended to open git issues on provider repos about the latest CAPI beta release.
+
+## Pre-requisites
+
+- Create a github token which has a access to the following scopes. And save it in your environment as `GITHUB_ISSUE_OPENER_TOKEN`
+  - `repo:status` - Grants access to commit status on public and private repositories.
+  - `repo_deployment` - Grants access to deployment statuses on public and private repositories.
+  - `public_repo` - Grants access to public repositories
+- Decide upon the title of the issue to be opened. Update it in the utility.
+- Update the issue body. Make sure to include following release and upcoming schedule.
+  
+  ```md
+  <!-- body -->
+  <!-- TODO: remove all TODOs before running this utility -->
+  <!-- TODO: update CAPI release semver -->
+  CAPI v1.x.0-beta.0 has been released and is ready for testing.
+  Looking forward to your feedback before CAPI 1.x.0 release!
+
+  ## For quick reference
+
+  <!-- body -->
+  <!-- TODO: CAPI release notes -->
+  - [CAPI v1.x.0-beta.0 release notes](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.x.0-beta.0)
+  - [Shortcut to CAPI git issues](https://github.com/kubernetes-sigs/cluster-api/issues)
+
+  ## Following are the planned dates for the upcoming releases
+
+  <!-- TODO: update CAPI release timeline -->
+  |Release|Expected Date|
+  |-----|-----|
+  |v1.5.0-beta.x | Tuesday 5th July 2023|
+  |release-1.5 branch created (Begin [Code Freeze])|Tuesday 11th July 2023|
+  |v1.5.0-rc.0 released|Tuesday 11th July 2023|
+  | release-1.5 jobs created | Tuesday 11th July 2023 |
+  | v1.5.0-rc.x released | Tuesday 18th July 2023 |
+  | v1.5.0 released | Tuesday 25th July 2023 |
+  <!-- body -->
+
+  <!-- [List of CAPI providers](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#communicate-beta-to-providers) -->
+  ```
+
+## How to run the tool
+
+- From the root of the project Cluster API, run below to run the utility.
+
+  ```sh
+  make provider-issues
+  ```
+
+- To dry-run the tool, navigate to the `provider-issues:` target in the top level [Makefile](../../../Makefile) and change `--dryRun=false` to `--dryRun=true`
```

Comment:
As above - would be good to have the make target default to dry-run and be set by an env variable.

Reply:
Updated.